mongo/integration/errors_test.go

```diff
@@ -4,6 +4,7 @@
 // not use this file except in compliance with the License. You may obtain
 // a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
 
+//go:build go1.13
```

Comment:
Is this intended to be included?

Reply:
See above above. Not intentional, but I think we should keep it.